support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/Pac4jInitialFlowSetupAction.java

```diff
@@ -0,0 +1,41 @@
+package org.apereo.cas.web.flow;
+
+import org.apereo.cas.authentication.AuthenticationServiceSelectionPlan;
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.services.ServicesManager;
+import org.apereo.cas.web.support.ArgumentExtractor;
+import org.apereo.cas.web.support.CookieRetrievingCookieGenerator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.webflow.execution.Event;
+import org.springframework.webflow.execution.RequestContext;
+
+import java.util.List;
+
+/**
+ * Class to set up variables at webflow action initialisation.
+ *
+ * @author Francis Le Coq
+ * @since 5.2
+ */
+public class Pac4jInitialFlowSetupAction extends InitialFlowSetupAction {
```

Comment:
Why is this necessary?

Reply:
Because on Pac4j you dont need configureCookieGenerators(context); configureWebflowContext(context); to be executed

Reply:
In that case, you need to simply add a flag that displays that behavior, rather than adding a whole new class and action and accompanying beans.

Reply:
Furthermore, as far as I can tell the configuration of cookie generators only sets paths for the generators. Therefore, is there a reason that operation is removed? does it cause a failure or break something? If not, you should simply let it be and remove this class altogether.